k8sdeps/transformer/patch/conflictdetector.go

```diff
@@ -105,7 +106,7 @@ func (smp *strategicMergePatch) findConflict(
 		if i == conflictingPatchIdx {
 			continue
 		}
-		if !patches[conflictingPatchIdx].OrgId().Equals(patch.OrgId()) {
+		if !patches[conflictingPatchIdx].OrgId().GvknEquals(patch.OrgId()) {
```

Comment:
We just changed it from GvknEquals to Equals because it fails in multi name space environment.

Reply:
Thanks for noticing this! Missed these unintended changes in the diff before I submitted the PR, fixed that.